framework/src/main/java/org/checkerframework/common/basetype/messages.properties

```diff
@@ -13,8 +13,8 @@ vector.copyinto.type.incompatible=incompatible component type in Vector.copyinto
 return.type.incompatible=incompatible types in return.%ntype of expression: %s%nmethod return type: %s
 annotation.type.incompatible=incompatible types in annotation.%nfound   : %s%nrequired: %s
 conditional.type.incompatible=incompatible types in conditional expression.%nfound   : %s%nrequired: %s
-type.argument.type.incompatible=incompatible types in type argument.%nfound   : %s%nrequired: %s
-argument.type.incompatible=incompatible types in argument.%nfound   : %s%nrequired: %s
+type.argument.type.incompatible=invalid type argument for type parameter %s of %s.%nfound   : %s%nrequired: %s
```

Comment:
I would rather keep this as `incompatible type argument for type parameter %s of %s...` and `incompatible argument for parameter %s of %s.` to stay consistent with all the other `type.incompatible` messages.